packages/modelviewer.dev/data/docs.json

```diff
@@ -907,6 +907,14 @@
           "<a href=\"../examples/scenegraph/#variants\">Related examples</a>"
         ]
       },
+      {
+        "name": "variantName",
+        "htmlName": "variantName",
+        "description": "Corresponding property of the attribute <code>variant-name</code>.",
```

Comment:
This is not necessary (you'll notice none  of our other attributes have property definitions, since all  HTML attributes always have corresponding properties). That said, I'm open to changing the overall  docs template to  make that more clear.

Reply:
Wasn't sure if this was omitted on purpose or not, omitting it does assume that anyone can work out the fact that `variant-name` corresponds to a property called `variantName`.